tests/unit/dataactbroker/test_domainRoutes.py

```diff
@@ -0,0 +1,51 @@
+import json
+from unittest.mock import Mock
+
+from flask import Flask
+import pytest
+
+from dataactbroker import domainRoutes
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.models.userModel import UserAffiliation
+from tests.unit.dataactcore.factories.domain import CGACFactory
+from tests.unit.dataactcore.factories.user import UserFactory
+
+
+@pytest.fixture
+def domain_app():
+    app = Flask('test-app')
+    domainRoutes.add_domain_routes(app)
+    app.teardown_appcontext(lambda exception: GlobalDB.close())
+    yield app.test_client()
```

Comment:
Could use dataactbroker.app:createApp here, but there's a lot of configurations there that I didn't want to mess with.